apps/web/components/Kbar.tsx

```diff
@@ -185,33 +195,40 @@ export const KBarContent = () => {
   return (
     <KBarPortal>
       <KBarPositioner>
-        <KBarAnimator className="bg-white shadow-lg">
-          <KBarSearch className="min-w-96 rounded-sm px-4 py-2.5 focus-visible:outline-none" />
+        <KBarAnimator className="z-10 w-full max-w-screen-sm overflow-hidden rounded-sm bg-white shadow-lg">
+          <div className="flex items-center justify-center border-b">
+            <SearchIcon className="mx-3 h-4 w-4 text-gray-500" />
+            <KBarSearch className="w-full rounded-sm py-2.5 focus-visible:outline-none" />
+          </div>
           <RenderResults />
+          <div className="flex items-center space-x-1 border-t px-2 py-1.5 text-gray-500">
+            <SwitchVerticalIcon className="h-4 w-4" /> <span className="pr-2 text-xs">Navigate</span>
+            <CornerDownLeft className="h-4 w-4" />
+            <span className="pr-2 text-xs">Open</span>
+            <Command className="h-3 w-3" />
+            <span className="pr-1 text-xs">+ K </span>
+            <span className="pr-2 text-xs">Close</span>
```

Comment:
Command wouldn't be the best for windows users, also we can close the bar using ESC key (I believe on both mac and windows) so maybe we can think about that?

Reply:
fixed